tests/pages/package-show.js

```diff
@@ -57,6 +57,42 @@ export default {
     ));
   },
 
+  clickCustomCoverageCancelButton() {
+    return convergeOn(() => {
+      expect($('[data-test-eholdings-package-details-cancel-custom-coverage-button]')).to.exist;
+    }).then(() => (
+      $('[data-test-eholdings-package-details-cancel-custom-coverage-button] button').click()
+    ));
+  },
+
+  clickCustomCoverageEditButton() {
+    return convergeOn(() => {
+      expect($('[data-test-eholdings-package-details-edit-custom-coverage-button]')).to.exist;
+    }).then(() => (
+      $('[data-test-eholdings-package-details-edit-custom-coverage-button] button').click()
+    ));
+  },
+
+  clickCustomCoverageSaveButton() {
+    return convergeOn(() => {
+      expect($('[data-test-eholdings-package-details-save-custom-coverage-button]')).to.exist;
+    }).then(() => {
+      return $('[data-test-eholdings-package-details-save-custom-coverage-button] button').click();
+    });
+  },
+
+  clickCustomCoverageAddButton() {
+    return convergeOn(() => {
+      expect($('[data-test-eholdings-package-details-custom-coverage-button] button')).to.exist;
+    }).then(() => (
+      $('[data-test-eholdings-package-details-custom-coverage-button] button').click()
+    ));
```

Comment:
we should probably extract these into a common click helper.  i'm not insisting on now, but soon.